subsys/net/lib/lwm2m/lwm2m_engine.h

```diff
@@ -27,6 +27,7 @@
 #define LWM2M_FORMAT_APP_OCTET_STREAM	42
 #define LWM2M_FORMAT_APP_EXI		47
 #define LWM2M_FORMAT_APP_JSON		50
+#define LWM2M_FORMAT_APP_SEML_JSON	110
```

Comment:
Typo, should be _LWM2M_FORMAT_APP_SEML_JSON_.

Reply:
ah this can be fixed at observation PR